test/IDE/complete_override_objc.swift

```diff
@@ -10,13 +10,21 @@ class Sub : FooClassBase {
   #^PA^#
 }
 
-// CHECK1: Begin completions, 8 items
+// CHECK1: Begin completions, 16 items
+// CHECK1: Decl[StaticMethod]/Super:           override class func fooBaseInstanceFunc0() {|}
 // CHECK1: Decl[InstanceMethod]/Super:         override func fooBaseInstanceFunc0() {|}
```

Comment:
This means every ObjC classes (`NSObject` subclasses) see duplicated candidates for override completion, one for class method, one for instance method.
@jrose-apple If we want to hide "AlsoImportAsClassMethod" methods, what is the best way to distinguish them?

Reply:
I…don't think there is one right now. We don't distinguish those methods from normal class methods on the Swift side. I guess we'll have to add a flag for that, or just live with it because it doesn't come up that much.

Reply:
Oh. I didn't notice it was the same method. What's the purpose of "AlsoImportAsClassMethod"?

Reply:
https://github.com/apple/swift/commit/69a51655c751d9c99137906cc97f4c164084054d explains that.

Reply:
Thanks for the explanation Jordan.
I think we can live with it for now. At least, we should not implement it in *this* PR.

Reply:
> [69a5165](https://github.com/apple/swift/commit/69a51655c751d9c99137906cc97f4c164084054d) explains that

@jrose-apple Could you educate me on the historical reason for adopting this relationship rather than having both instance and class methods where necessary?